algorithms/rainbow/main.py

```diff
@@ -0,0 +1,177 @@
+"""
+Adapted from https://github.com/Kaixhin/Rainbow
+
+The main file needed within rainbow. Runs of the train and test functions from their respective
+files.
+
+Example of use:
+`cd algorithms/rainbow`
+`python main.py`
+
+Runs Rainbow DQN on our AI2ThorEnv wrapper with default params. Optionally it can be run on any
+atari environment as well using the "game" flag, e.g. --game space_invaders.
```

Comment:
is it not `SpaceInvaders-v0` ?

Reply:
Maybe choose another example like breakout-v4 or something?

Reply:
Chose seaquest